code/pizzabot.js

```diff
@@ -1,7 +1,50 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian";
+const hawaiian = "Hawaiian";
+const pepperoni = "Pepperoni";
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+const greet = (`Hey! Happy to serve your pizza! <br> On our menue we have <span style="color:#fa3e54">${vegetarian}</span>, <span style="color:#4c3ffc">${hawaiian}</span> and <span style="color:#059e7b">${pepperoni}</span> pizzas.`);
+document.getElementById("welcome").innerHTML = greet;
 
-//Put your Javscript code here:
+let clicked = () => {
```

Comment:
You can use `const` to these functions!
```suggestion
const clicked = () => {
```